### PR TITLE
Hardcode maker details

### PR DESCRIPTION
**Pull request by klochowicz**
Maker pubkey is now hard-coded and matches the checked in seed.

See commit description for further info.

### CODE REVIEW

**Comment by klochowicz** [comment]
this PR was an outcome of discussions around #209 - to go as fast as we can, we can just hardcode the counterparty details.

File: maker/src/main.rs [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-use std::env::temp_dir;
+use std::env::current_dir;
 use std::time::Duration;
 use ten_ten_one::wallet;
 use tracing::metadata::LevelFilter;
 
 #[rocket::main]
 async fn main() -> Result<()> {
-    let path = temp_dir();
+    let path = current_dir()?.join("data").join("maker");
```
Comment (bonomat) [inline maker/src/main.rs:11]: much better :) 
Note: we will need to be able to pass this in as an arg once we want to run it in a container.

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,5 @@ bridge_generated.h
 #ignore keystores
 *.jks
 
+# Artefacts from running maker binary
+/data/maker/
```
Comment (bonomat) [inline .gitignore:19]: 🙃
I love this comment: missing new line

**Review by bonomat — APPROVED** [review]
LGTM

File: maker/src/main.rs (hunk 1 above)
Comment (klochowicz) [reply maker/src/main.rs:11]: yes, `$DATA_DIR` and `$NETWORK` should soon be added from `clap` :)

**Review by da-kami — APPROVED** [review]
Happy to merge this part! 
Once that is in we should also change `open_channel` API to use these values rather than passing the connection string from the user interface.
We will then need an API endpoint to request the "current" maker from the rust code.

File: .gitignore (hunk 2 above)
Comment (klochowicz) [reply .gitignore]: added :)

**Comment by klochowicz** [comment]
> Happy to merge this part!
> Once that is in we should also change `open_channel` API to use these values rather than passing the connection string from the user interface.
> We will then need an API endpoint to request the "current" maker from the rust code.

great idea! I'll continue working on that straight away.

**Comment by klochowicz** [comment]
> Happy to merge this part!
> Once that is in we should also change `open_channel` API to use these values rather than passing the connection string from the user interface.
> We will then need an API endpoint to request the "current" maker from the rust code.

addressed in #214

**Pull request merged**